.github/workflows/tests.yml

```diff
@@ -87,18 +87,9 @@ jobs:
     - name: Get changed files
       id: files
       uses: tj-actions/changed-files@v34
-
-    - name: Filter changed files
-      id: python-files
-      run: |
-        for file in ${{ steps.files.outputs.all_changed_files }}; do
-          if [[ $file == *.py ]]; then
-            if [[ $file != test/* ]]; then
-              changed_files="$changed_files $file"
-            fi
-          fi
-        done
-        echo "changed_files=$changed_files" >> $GITHUB_OUTPUT
+      with:
+        files: |
+          *.py
```

Comment:
I think we can ignore files under `test/` to make the job even faster, see https://github.com/deepset-ai/haystack/pull/3930/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957f

Reply:
Right, I forgot. Adding that.